exporter/README.md

```diff
@@ -58,7 +58,7 @@ The following information will be stored and served by exporter:
     "message": {
       "type": 3,
       "round": 1,
-      "lambda": "...",
+      "identifier": "...",
```

Comment:
make sure this is really the right key

Reply:
@nivBlox it is, but the whole example is outdated, we don't have `seq_number` or `signer_ids` anymore and they are not the scope of this PR. I think that needs to be fixed in a PR for making sure that docs are up-to-date.